dirty_cat/super_vectorizer.py

```diff
@@ -491,11 +490,15 @@ def get_feature_names_out(self, input_features=None) -> List[str]:
         return all_trans_feature_names
     
     def get_feature_names(self) -> List[str]:
-        """ Deprecated, use "get_feature_names_out"
         """
-        warn(
-            "get_feature_names is deprecated in scikit-learn > 1.0. "
-            "use get_feature_names_out instead",
-            DeprecationWarning,
+        Ensures compatibility with sklearn < 1.0.
+        Use `get_feature_names_out` instead.
+        """
+        if Version(sklearn_version) >= '1.0':
+            warn(
+                "Following the changes in scikit-learn 1.0, "
+                "get_feature_names is deprecated. "
+                "Use get_feature_names_out instead.",
+                DeprecationWarning
```

Comment:
stacklevel also (though actually the initial call did not have stacklevel, so I'm asking for an improvement.